tests/e2e/specs/checkoutPage_spec.js

```diff
@@ -0,0 +1,195 @@
+describe('CheckoutPage', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  function shouldShowCorrectCartLike() {
```

Comment:
```suggestion
  function shouldAllowToPlaceOrders() {
```

I forgot to change this test name 😓

Reply:
And that was not the test name, so it was actually correct 😅
Reverted ⬇️